Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV NODE_VERSION "v12.18.1"
```

Comment:
The Node version inside Electron is v12.13.0. It helps if the versions match, otherwise the native dependencies need to be recompiled.

Reply:
Agreed and done.